.github/workflows/dockerimage.yml

```diff
@@ -78,7 +78,7 @@ jobs:
         grep --fixed-strings '<p>It works!</p>' /tmp/h2
 
 
-        docker run --rm --network host ymuski/curl-http3 \
+        docker run --rm --network host ghcr.io/macbre/curl-http3 \
           curl -v https://localhost:8889 --http3 2>&1 | tee /tmp/h3
```

Comment:
```suggestion
          curl -v --insecure https://localhost:8889 --http3 2>&1 | tee /tmp/h3
```

Otherwise, we get:

```
* SSL certificate problem: unable to get local issuer certificate
* connect to 127.0.0.1 port 8889 failed: SSL peer certificate or SSH remote key was not OK
```